.gitlab-ci.yml

```diff
@@ -22,7 +22,8 @@ test_backend:
   script:
     - docker-compose -f docker-compose.tests.yml up -d
     - sleep 15
-    - docker-compose exec -T backend pytest
+    - docker-compose exec -T backend pytest -p no:warnings --cov=. --cov-report html
+    - docker-compose exec -T backend coverage report --omit=things_that_arent_mine/* --precision=2
```

Comment:
czy to jest potrzebne ? generujemy report wyżej

Reply:
To nam printuje i CI pobiera według regex'a podanego w panelu gitlab

Reply:
A co dokładnie oznacza tutaj 
--omit=things_that_arent_mine/*